### PR TITLE
Refactor home screen tests

### PR DESCRIPTION
**Pull request by webteckie**
## Description of changes

Refactored the home screen tests to make use of the latest home screen page object API.

## Related issues (if any)
N/A

## Testing

- [X] Please confirm `npm run test-all` ran successfully.

### CODE REVIEW

File: test/e2e/adminUI/tests/group003Home/uiTestHomeView.js [hunk 1]
```diff
@@ -5,36 +5,42 @@ module.exports = {
...
-	},
-	'Home view should have tabs under Access dashboard group': function (browser) {
-		browser.adminUIHomeScreen.section.accessGroup.assertUI();
-	},
-	'Home view should have tabs under Fields dashboard group': function (browser) {
-		browser.adminUIHomeScreen.section.fieldsGroup.assertUI();
-	},
-	'Home view should have tabs under Other dashboard group': function (browser) {
-		browser.adminUIHomeScreen.section.otherGroup.assertUI();
+		browser.adminUIHomeScreen.assertElementIsVisible({ element: 'dashboardHeader' });
```
Comment (jstockwin) [inline test/e2e/adminUI/tests/group003Home/uiTestHomeView.js]: @webteckie We've lost the `.text.to.equal('e2e')` assertion, was that a consious decision to drop it?

**Comment by jstockwin** [comment]
@webteckie One comment, otherwise looks good provided that tests pass.

File: test/e2e/adminUI/tests/group003Home/uiTestHomeView.js (hunk 1 above)
Comment (webteckie) [inline test/e2e/adminUI/tests/group003Home/uiTestHomeView.js]: @jstockwin yeah, somehow did...will take care of it at some point.

Comment (jstockwin) [inline test/e2e/adminUI/tests/group003Home/uiTestHomeView.js]: @webteckie Do you mean you'll take care of it in this PR, or another time?

Comment (webteckie) [inline test/e2e/adminUI/tests/group003Home/uiTestHomeView.js]: @jstockwin I don't have time now...feel free to merge and take care of it.  Or just oush the change to the PR.

**Comment by JedWatson** [comment]
@webteckie @jstockwin this looks good to merge but has a conflict in the package.json. Want me to clear that up and get this into master?

**Comment by jstockwin** [comment]
@JedWatson yes please, although my comments haven't been addressed yet and
I don't have time. If you could add a TODO in the code saying we need to re
implement the 'e2e' text assertion that'd be really helpful.

**Comment by JedWatson** [comment]
Done!

**Pull request merged**